runtime/oti/cfr.h

```diff
@@ -921,7 +921,8 @@ typedef struct J9CfrClassFile {
 
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
 #define CFR_CLASS_ACCESS_MASK					(CFR_ACC_PUBLIC | CFR_ACC_FINAL | CFR_ACC_IDENTITY | CFR_ACC_INTERFACE | CFR_ACC_ABSTRACT | CFR_ACC_SYNTHETIC | CFR_ACC_ANNOTATION | CFR_ACC_ENUM)
-#define CFR_FIELD_ACCESS_MASK  					(CFR_ACC_PUBLIC | CFR_ACC_PRIVATE | CFR_ACC_PROTECTED | CFR_ACC_STATIC | CFR_ACC_FINAL | CFR_ACC_VOLATILE | CFR_ACC_TRANSIENT | CFR_ACC_SYNTHETIC | CFR_ACC_ENUM | CFR_ACC_STRICT_INIT)
+/* Renaming CFR_ACC_STRICT to CFR_ACC_STRICT_INIT will break DDR in Java 8. */
```

Comment:
I think it's more about "using" than "renaming". I think it should say something like "using CFR_ACC_STRICT_INIT is more appropriate than using CFR_ACC_STRICT, but doing so would break DDR in IBM Java 8" - note the additional qualification of "IBM" here, Semeru builds are fine, even for Java 8.